diff --git a/node_modules/@devexpress/dx-react-grid-material-ui/dist/dx-react-grid-material-ui.d.ts b/node_modules/@devexpress/dx-react-grid-material-ui/dist/dx-react-grid-material-ui.d.ts
index 31d805e..f508bfe 100644
--- a/node_modules/@devexpress/dx-react-grid-material-ui/dist/dx-react-grid-material-ui.d.ts
+++ b/node_modules/@devexpress/dx-react-grid-material-ui/dist/dx-react-grid-material-ui.d.ts
@@ -153,7 +153,7 @@ export namespace Grid {
   export type RootProps = GridBase.RootProps;
 }
 
-export interface GridProps {
+export type GridProps = React.PropsWithChildren<{
   /** An array containing custom data. A user defines the access to this data. Refer to Data Accessors for details. */
   rows: ReadonlyArray<any>;
   /** Specifies for which row fields columns are created. */
@@ -164,7 +164,7 @@ export interface GridProps {
   getCellValue?: (row: any, columnName: string) => any;
   /** A component that renders the grid root layout. */
   rootComponent?: React.ComponentType<GridBase.RootProps>;
-}
+}>
 
 /** The Grid is a root container component designed to process and display data specified via the `rows` property. You can use the `columns` property to configure columns.The Grid's functionality  is implemented in several plugins specified as child components.The Grid is a PluginHost and can contain only other plugins and plugin primitives. */
 export declare const Grid: React.ComponentType<GridProps> & {
